flink-mesos/src/main/scala/org/apache/flink/mesos/scheduler/LaunchCoordinator.scala

```diff
@@ -66,10 +66,14 @@ class LaunchCoordinator(
       .withLeaseRejectAction(new Action1[VirtualMachineLease]() {
         def call(lease: VirtualMachineLease) {
           LOG.info(s"Declined offer ${lease.getId} from ${lease.hostname()} "
-            + s"of ${lease.memoryMB()} MB, ${lease.cpuCores()} cpus.")
+            + s"of memory ${lease.memoryMB()} MB, ${lease.cpuCores()} cpus, ${lease.getScalarValue("gpus")} gpus, "
+            + s"of disk: ${lease.diskMB()} MB, network: ${lease.networkMbps()} Mbps")
```

Comment:
just FYI in our setup fenzo doesn't work with the network, since this resource has different naming. Taking into about that our network isolation is being decommissioned we can skip merging of this implementation

Reply:
yeah I noticed that we're also not passing in the network resource requirements. I ended up adding the line to make it consistent as we were logging details on the other resources.

Reply:
> Taking into about that our network isolation is being decommissioned

Actually the isolation part is being temporarily removed but the resource itself still exist so this change is relevant.